src/app/src/components/SavedVisualizations.js

```diff
@@ -71,8 +71,18 @@ const SavedVisualizations = () => {
     return (
         <Box>
             <Breadcrumbs />
-            <Flex bg='white' p={4} border='1px solid rgb(222, 227, 233)'>
+            <Flex
+                bg='white'
+                p={4}
+                border='1px solid rgb(222, 227, 233)'
+                align='baseline'
+            >
                 <Text fontSize='2xl'>Saved Charts</Text>
+                <Spacer />
+                <Text>
+                    Charts saved on your browser using cookies. Deleting browser
```

Comment:
While this is what was requested, I'm afraid referencing cookies will be confusing since clearing them won't actually clear the localStorage.

What about something more along the lines of
> Charts are saved locally in your browser. Clearing your browser data may impact this page.

Reply:
Thanks for the review! Changes made in 76f55cc to saved charts language, and route updated. Most of the places where we link to the charts route we reference the `ROUTES` constant, but I updated to use that constant in the places where it was hard-coded.